tests/consensus_tests/test_failed_snapshot_recovery.py

```
@@ -167,9 +167,12 @@ def is_qdrant_started() -> bool:
     except Exception as e:
         raise Exception(f"Qdrant did not start in time after recovering corrupt snapshot, maybe it crashed: {e}")
 
-    # Assert storage contains initialized flag after restart (this means a dummy replica is loaded)
     flag_path = shard_initializing_flag(peer_dirs[-1], COLLECTION_NAME, 0)
-    assert os.path.exists(flag_path)
+    flag_exists = os.path.exists(flag_path)
```

Ruff (0.8.2)

`os` may be undefined, or defined from star imports

(F405)